lib/miq_expression/field.rb

```diff
@@ -90,6 +94,10 @@ def column_type
     end
   end
 
+  def arel_attribute
```

Comment:
@kbrock would be great if it could be avoided. I don't see it as effectively public - our interface is merely consistent with a lot of arel, providing `eq`, `not_eq`, etc.. It may be a good idea to break this class up if it's losing cohesion (but not as part of this PR)